v6/emoji/emoji.py

```diff
@@ -37,13 +38,909 @@ class Plugin(RestExtension):
     def set_site(self, site):
         self.site = site
         roles.register_local_role('emoji', emoji_role)
+        if hasattr(site, 'transforms'):
+            site.transforms.append(Emojis)
+        else:
+            site.transforms = [Emojis]
 
 
 def emoji_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
     text = text.lower()
+    LOGGER.warn('The role :emoji:`{0}` is deprecated. Use |{0}| instead'.format(text))
```

Comment:
because *There should be one-- and preferably only one --obvious way to do it.* (from the Zen)